docs/design/docs/docs-named-styles.md

```
@@ -29,7 +29,10 @@ Roadmap item: Phase **6.5 Named Styles** in
 - A document-level style registry stored in the Yorkie CRDT (root level,
   beside `pageSetup`) holding **only overrides** of built-in style defs.
 - "Update '<style>' to match" — redefine a style from the caret block's
-  current formatting; all blocks of that style reflow.
+  current formatting, minus whatever that block merely *inherited* from the
+  style (see
+  ["Update to match" must not capture defaults](#update-to-match-must-not-capture-defaults));
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)